src/main/java/dividing/Starter.java

```diff
@@ -0,0 +1,21 @@
+package dividing;
+
+import java.util.Scanner;
+
+
+public class Starter  {
+
+    public static void main(String[] args) {
```

Comment:
Please check formatting of this method. There is a bunch of missing white spaces

Reply:
Done